lib/browser/desktop-capturer.js

```diff
@@ -1,23 +1,23 @@
 'use strict'
 
-const ipcMain = require('electron').ipcMain
-const desktopCapturer = process.atomBinding('desktop_capturer').desktopCapturer
+const {ipcMain} = require('electron')
+const {desktopCapturer} = process.atomBinding('desktop_capturer')
 
-var deepEqual = function (opt1, opt2) {
-  return JSON.stringify(opt1) === JSON.stringify(opt2)
-}
+const deepEqual = (a, b) => JSON.stringify(a) === JSON.stringify(b)
 
 // A queue for holding all requests from renderer process.
-var requestsQueue = []
+let requestsQueue = []
+
+const electronSources = 'ELECTRON_BROWSER_DESKTOP_CAPTURER_GET_SOURCES'
+const capturerResult = (id) => `ELECTRON_RENDERER_DESKTOP_CAPTURER_RESULT_${id}`
 
-ipcMain.on('ELECTRON_BROWSER_DESKTOP_CAPTURER_GET_SOURCES', function (event, captureWindow, captureScreen, thumbnailSize, id) {
-  var request
-  request = {
-    id: id,
+ipcMain.on(electronSources, (event, captureWindow, captureScreen, thumbnailSize, id) => {
+  const request = {
+    id,
     options: {
-      captureWindow: captureWindow,
-      captureScreen: captureScreen,
-      thumbnailSize: thumbnailSize
+      captureWindow,
+      captureScreen,
+      thumbnailSize
```

Comment:
I don't know if linter would like it or not,
but diff-wise I would love to see a trailing comma here )

Reply:
the linter gets mad about that, sadly :(

Reply:
The linter config should be changed then )

Reply:
i agree on this, i would prefer it too

Reply:
Jumping on the "linter plz allow trailing commas" bandwagon here 😆